benchmarks/aes/README.md

```diff
@@ -0,0 +1,65 @@
+# AES Benchmark
```

Comment:
please mention the relay container

Reply:
We have just started work on that, so I think I should leave this as is?

Reply:
then that has to be merged before this PR

Reply:
I would need to test the relay using some function. If AES is there it'll make everything easier (instead of manually copying all the commit changes). Since it already works well with helloworld, no broken functionality would be added either. If possible, please merge at least AES, then we can make another PR to add relay.

Reply:
@ustiugov I dont see the point this PR depend on the relay container...
For now the aes function implements the helloworld interface.
I know it makes sense to add the relay also to the this function but then you should also update your existing functions with that...
So if it makes it easier for Harshit why not doing it stepwise?

Reply:
the existing functions should be fine... ok for now